<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBoxView">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>221</width>
     <height>631</height>
    </rect>
   </property>
   <property name="title">
    <string>实时变量</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" colspan="2">
     <widget class="QPushButton" name="ButtonViewSave">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QTableWidget" name="tableWidgetView">
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>88</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxTcp">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>331</width>
     <height>102</height>
    </rect>
   </property>
   <property name="title">
    <string>TCP</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="labelIP">
      <property name="text">
       <string>IP: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelPort">
      <property name="text">
       <string>端口:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditPort">
      <property name="text">
       <string>8888</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="ButtonConnect">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditIP">
      <property name="text">
       <string>192.168.43.150</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="ButtonDisConnect">
      <property name="text">
       <string>断开</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxImg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>实时图像</string>
   </property>
   <widget class="QLabel" name="labelImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>280</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>消息</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="toolTip">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxMatch">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>331</width>
     <height>137</height>
    </rect>
   </property>
   <property name="title">
    <string>比赛</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditUnloadx">
      <property name="text">
       <string>-1.81</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditLoadx">
      <property name="text">
       <string>1.89</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelWait">
      <property name="text">
       <string>等待区：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEditLoady">
      <property name="text">
       <string>-4.60</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEditWaity">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelUnload">
      <property name="text">
       <string>卸货区：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelLoad">
      <property name="text">
       <string>装货区：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="ButtonToLoad">
      <property name="text">
       <string>前往装货区</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditUnloady">
      <property name="text">
       <string>-6.025</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="ButtonToWait">
      <property name="text">
       <string>前往等待区</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditWaitx">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="ButtonToUnload">
      <property name="text">
       <string>前往卸货区</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ButtonStop">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停车</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonStart">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>启动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRestart">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
